User (#2512)
umbraco/UmbracoDocs: Umbraco-StarterKit/Manage-users/index.md

# Manage users

In this section, you can find all the information and details you need in order to work with and manage both users and members on your website.

## What is a User

A user is someone who has access to the Umbraco backoffice and is not to be confused with Members. When you enter your Umbraco project you will find that a user allready excist made with the login (email) and password entered during creation of the project. Users can be created, edited, and managed in the User section.
Managing your users.
You can manage your users so they can’t publish or edit only read, or you can disable someone from your project if they should not have access anymore.

> Please note that for now a user cannot be deleted, they can only be disabled

Reviewer: ```suggestion
A user cannot be deleted, they can only be disabled
```

The reason for this is that, if the user were deleted then everything they created would be deleted along with the user.

Please see the [User](../Manage-users/Users/index.md) section for more information about users and what you can do with them.

> :::note You will find that your project has a user named themeowner. It is there because this is what implements the entire theme on your project. If you remove this user your project won't have access to the theme anymore and that will make your project cease to function:::

Reviewer: ```suggestion
:::note 
You will find that your project has a user named themeowner. It is there because this is what implements the entire theme on your project. 
:::
```

# Manage users

In this section, you can find all the information and details you need in order to work with and manage both users and members on your website.

## What is a User

> A user is someone who has access to the Umbraco backoffice and is not to be confused with Members. When Umbraco has been installed a user will automatically be generated with the login (email) and password entered during installation. Users can be created, edited, and managed in the User section.

Reviewer: "When Umbraco has been installed a user will automatically be generated with the login (email) and password entered during installation."
Consider whether this is relevant to know, when working with Umbraco Uno.

Author: i feel this is relevant because we are working with people who don't necessarily know this

Reviewer: Alright, but they don't actually install Umbraco themselves, which is what this sentence is referencing. Perhaps instead it should say something about how a user is created using the credentials you entered when you took a trial? Or that it's based on your existing Cloud user? 🤔

> Managing your users.

Reviewer: Is this supposed to be a heading, or part of the sentence below?

Author: fixing it now

You can manage your users so they can’t publish or edit only read, or you can disable someone from your project if they should not have access anymore.

> Please note that for now a user cannot be deleted, they can only be disabled.

Reviewer: Maybe explain this a bit more.

> You will find that your project has a user named themeowner, it is there to make your project work.

Reviewer: Perhaps we can put this in a note, tip or warning box, to highlight it a bit more?

And I think we also need to explain it in more detail, why that user needs to be there 🤔

> Please see the [User](../Manage-users/Users/index.md) section for more information about users and what you can do with them.

Reviewer: ```suggestion
Please see the [User](../Manage-users/Users/index.md) article for more information about users and what you can do with them.
```

## What is a Member

> A member is someone who is strictly limited to the front end and typically these members will be customers, clients, or users of your page's front end.

Reviewer: ```suggestion
A member is someone who is strictly limited to the frontend and typically these members will be customers, clients, or users of your page's frontend.
```

> If you want to lern more about how you can manage your [Members](../Manage-users/Members/index.md) Theres an entire article about it.

Reviewer: ```suggestion
If you want to learn more about how you can manage your [Members](../Manage-users/Members/index.md) there's an entire article about it.
```

> ## What is a Team Member
> 
> Invite member in the portal, is a bit of weird one, as you don't invite a member but a user.
> While this can be a bit confusing, you can view a team member as a user of the backoffice.
> When you invite a team member you can decide if they need admin access, if you grant them that they will be able to access and edit everything.

Reviewer: ```suggestion
 
```
